packages/desktop-client/src/components/sidebar/Accounts.tsx

```
@@ -61,108 +57,129 @@ export function Accounts({
-            marginBottom: 5,
-          }}
-        />
-      )}
+        {budgetedAccounts.map((account, i) => (
+          <Account
+            key={account.id}
+            name={account.name}
+            account={account}
+            connected={!!account.bank}
+            pending={syncingAccountIds.includes(account.id)}
+            failed={failedAccounts && failedAccounts.has(account.id)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)